### DISCUSSION TITLE
How can I add startpage engine in flow.launcher?

### DISCUSSION THREAD

**Discussion by MihneaXB** [Flow-Launcher/Flow.Launcher#2340] [Q&A]
Hello, I use startpage as my main search browser, and I want to add a shortcut in flow.launcher, but I can't figure it out. Can someone help me in this manner?

**Comment by BrianBlowtasky** [comment]
https://www.startpage.com/sp/search?q={q}

**Reply by jjw24** [reply]
Just adding to this, go to settings -> plugins -> web search plugin, then add above URL.
